stable/insights-agent/ci/test-values.yaml

```diff
@@ -149,6 +149,19 @@ awscosts:
   tagvalue: staging.internal.reactiveops.com
   workgroup: cur_athena_workgroup
 
+cloudcosts:
```

Comment:
I wonder if we want to have a provider specific block. i.e.: for `aws`
```
cloudcosts:
  enabled: true
  aws:
    accessKey: xyz
    secretAccess: xyz
    ....
```

Reply:
yeah that makes sense